docs/ADL2/master07.13-adl_terminology.adoc

```diff
@@ -6,9 +6,45 @@ This section describes the syntax of the `terminology` section of an archetype.
 
 The `terminology` section of an archetype is expressed in ODIN, and is where codes representing node identifiers, constraints on coded term values, and bindings to terminologies are defined. Linguistic language translations are added in the form of extra blocks keyed by the relevant language. The following example shows the general layout of this section.
 
+[tabs,sync-group-id=adl-example]
+====
+at-coded ADL2::
++
 [source, adl]
 --------
-terminology 
+terminology
+    term_definitions = <
+        ["en"] = <
+            ["at0001"] = <...>
+            ["at0002"] = <...>
+            ["ac1"] = <...>
```

Comment:
I'm still confused about why those ac codes remain in at examples. :)

Reply:
Ac codes are for valuesets. Since adl2 defines valueset in the terminology section 'not' inline. We still need those. Even if we don't use id codes. 
Ac codes will be a set/list of at codes. 

Ac codes will not end up in the data I expect. So it's not relevant for backwards compatibility of data. And it's much nicer to define them separately than inline imho. 
Any objections?